### ISSUE TITLE
Short URLS need `indices:data/write/bulk` perms on cluster

### ISSUE THREAD

**Issue by matejzero** [floragunncom/search-guard#451]
I had a problem where users couldn't generate short URLs. We were getting an error: 

```
Url Shortener: Error 403 Forbidden: no permissions for [indices:data/write/bulk] and User [name=matejz, roles=[CN=...]] 
```

My roles for `matejz` user have needed permissions:
```
sg_role_it:
  cluster:
    - CLUSTER_COMPOSITE_OPS_RO
    - indices:data/read/scroll*
  indices:
    'universal':
      '*':
        - KIBANA_READ
    '*':
      '*':
        - KIBANA_READ
        - MONITOR
    '?kibana*':
      '*':
        - INDICES_ALL
```

Nonetheless, it still didn't work. Looking at the logs, this caught my attention:
```
[2018-02-14T12:09:51,381][DEBUG][c.f.s.c.PrivilegesEvaluator]   resolved cluster actions:[indices:admin/aliases/exists*, indices:data/read/msearch, indices:data/read/scroll*, indices:data/read/coordinate-msearch*, indices:admin/aliases/get*, indices:data/read/mget, indices:data/read/mtv]
[2018-02-14T12:09:51,381][DEBUG][c.f.s.c.PrivilegesEvaluator]   not match found a match for 'sg_role_it' and indices:data/write/bulk, check next role
```

I added `indices:data/write/bulk` permissions to cluster section of role config and it started to work, but that seems like a lot of permissions for such a simple use case (also, does that allow users to write to all indices??). 

Anyone else having such problems? 

ES: 5.6.6
SG: 5.6.6-18
JVM: java-1.8.0-openjdk-1.8.0.151-5.b12.el7_4.x86_64
OS: CentOS 7

Full ES debug output:
```
[2018-02-14T12:09:51,374][DEBUG][c.f.s.a.BackendRegistry  ] Try to extract auth creds from http basic
[2018-02-14T12:09:51,374][DEBUG][c.f.s.a.BackendRegistry  ] User 'matejz' is in cache? true (cache size: 7)
[2018-02-14T12:09:51,374][DEBUG][c.f.s.a.BackendRegistry  ] User 'User [name=matejz, roles=[CN...]]' is authenticated
[2018-02-14T12:09:51,374][DEBUG][c.f.s.a.BackendRegistry  ] sg_tenant 'null'
[2018-02-14T12:09:51,378][DEBUG][c.f.s.a.BackendRegistry  ] Try to extract auth creds from http basic
[2018-02-14T12:09:51,378][DEBUG][c.f.s.a.BackendRegistry  ] User 'matejz' is in cache? true (cache size: 7)
[2018-02-14T12:09:51,378][DEBUG][c.f.s.a.BackendRegistry  ] User 'User [name=matejz, roles=[CCN...]]' is authenticated
[2018-02-14T12:09:51,378][DEBUG][c.f.s.a.BackendRegistry  ] sg_tenant 'null'
[2018-02-14T12:09:51,378][DEBUG][c.f.s.c.PrivilegesEvaluator] ### evaluate permissions for User [name=matejz, roles=[CN...]] on vevnica
[2018-02-14T12:09:51,378][DEBUG][c.f.s.c.PrivilegesEvaluator] requested indices:data/write/index from 10.10.10.1:51506
[2018-02-14T12:09:51,378][DEBUG][c.f.s.c.PrivilegesEvaluator] Resolve [.kibana] from class org.elasticsearch.action.index.IndexRequest
[2018-02-14T12:09:51,378][DEBUG][c.f.s.c.PrivilegesEvaluator] indicesOptions IndicesOptions[id=48, ignore_unavailable=false, allow_no_indices=false, expand_wildcards_open=false, expand_wildcards_closed=false, allow_alisases_to_multiple_indices=false, forbid_closed_indices=true]
[2018-02-14T12:09:51,378][DEBUG][c.f.s.c.PrivilegesEvaluator] 1 raw indices [.kibana]
[2018-02-14T12:09:51,378][DEBUG][c.f.s.c.PrivilegesEvaluator] No date math indices found
[2018-02-14T12:09:51,378][DEBUG][c.f.s.c.PrivilegesEvaluator] Resolved [.kibana] to [.kibana]
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator] pre final indices: [.kibana]
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator] pre final types: [url]
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator] requested resolved indextypes: [IndexType [index=.kibana, type=url]]
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator] mapped roles for matejz: [sg_role_it]
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator] ---------- evaluate sg_role: sg_role_it
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator]   Resolve and match universal
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator]   resolved permitted aliases indices for universal: [universal]
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator]   matches for universal, will check now types [*]
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator] For index universal remaining requested indextype: [IndexType [index=.kibana, type=url]]
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator]   Try wildcard match for *
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator]   Wildcard match for *: [.kibana]
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator]   matches for *, will check now types [*]
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator]     match requested action [indices:data/write/index] against */*: [indices:*, indices:monitor/*]
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator]     removed .kibana*
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator] For index * remaining requested indextype: []
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator] found a match for 'sg_role_it.*', evaluate other roles
[2018-02-14T12:09:51,379][DEBUG][c.f.s.c.PrivilegesEvaluator]   Try wildcard match for ?kibana*
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator]   Wildcard match for ?kibana*: [.kibana]
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator]   matches for ?kibana*, will check now types [*]
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator]     match requested action [indices:data/write/index] against ?kibana*/*: [indices:*]
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator]     no match .kibana* in []
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator] For index ?kibana* remaining requested indextype: []
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator] found a match for 'sg_role_it.?kibana*', evaluate other roles
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator] Added to leftovers sg_role_it=>[]
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator] ### evaluate permissions for User [name=matejz, roles=[CN...]] on vevnica
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator] requested indices:data/write/bulk from 10.10.10.1:51506
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator] Resolve [.kibana] from class org.elasticsearch.action.index.IndexRequest
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator] indicesOptions IndicesOptions[id=48, ignore_unavailable=false, allow_no_indices=false, expand_wildcards_open=false, expand_wildcards_closed=false, allow_alisases_to_multiple_indices=false, forbid_closed_indices=true]
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator] 1 raw indices [.kibana]
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator] No date math indices found
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator] Resolved [.kibana] to [.kibana]
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator] pre final indices: [.kibana]
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator] pre final types: [url]
[2018-02-14T12:09:51,380][DEBUG][c.f.s.c.PrivilegesEvaluator] requested resolved indextypes: [IndexType [index=.kibana, type=url]]
[2018-02-14T12:09:51,381][DEBUG][c.f.s.c.PrivilegesEvaluator] mapped roles for matejz: [sg_role_it]
[2018-02-14T12:09:51,381][DEBUG][c.f.s.c.PrivilegesEvaluator] ---------- evaluate sg_role: sg_role_it
[2018-02-14T12:09:51,381][DEBUG][c.f.s.c.PrivilegesEvaluator]   resolved cluster actions:[indices:admin/aliases/exists*, indices:data/read/msearch, indices:data/read/scroll*, indices:data/read/coordinate-msearch*, indices:admin/aliases/get*, indices:data/read/mget, indices:data/read/mtv]
[2018-02-14T12:09:51,381][DEBUG][c.f.s.c.PrivilegesEvaluator]   not match found a match for 'sg_role_it' and indices:data/write/bulk, check next role
[2018-02-14T12:09:51,381][INFO ][c.f.s.c.PrivilegesEvaluator] No cluster-level perm match for User [name=matejz, roles=[CN...]] [IndexType [index=.kibana, type=url]] [Action [[indices:data/write/bulk]]] [RolesChecked [sg_role_it]]
[2018-02-14T12:09:51,381][INFO ][c.f.s.c.PrivilegesEvaluator] No permissions for {}
[2018-02-14T12:09:51,381][DEBUG][c.f.s.f.SearchGuardFilter] no permissions for [indices:data/write/bulk]
[2018-02-14T12:09:51,381][DEBUG][r.suppressed             ] path: /.kibana/url/9be293fa1c6a2b8d18e3507640115602/_create, params: {refresh=wait_for, index=.kibana, id=9be293fa1c6a2b8d18e3507640115602, op_type=create, type=url}
org.elasticsearch.ElasticsearchSecurityException: no permissions for [indices:data/write/bulk] and User [name=matejz, roles=[CN...]]
  at com.floragunn.searchguard.filter.SearchGuardFilter.apply(SearchGuardFilter.java:143) ~[?:?]
  at org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:168) ~[elasticsearch-5.6.6.jar:5.6.6]
  at org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:142) ~[elasticsearch-5.6.6.jar:5.6.6]
  at org.elasticsearch.action.bulk.TransportSingleItemBulkWriteAction.doExecute(TransportSingleItemBulkWriteAction.java:69) ~[elasticsearch-5.6.6.jar:5.6.6]
  at org.elasticsearch.action.bulk.TransportSingleItemBulkWriteAction.doExecute(TransportSingleItemBulkWriteAction.java:44) ~[elasticsearch-5.6.6.jar:5.6.6]
  at org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:170) ~[elasticsearch-5.6.6.jar:5.6.6]
  at com.floragunn.searchguard.filter.SearchGuardFilter.apply(SearchGuardFilter.java:138) ~[?:?]
  at org.elasticsearch.action.support.TransportAction$RequestFilterChain.proceed(TransportAction.java:168) ~[elasticsearch-5.6.6.jar:5.6.6]
  at org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:142) ~[elasticsearch-5.6.6.jar:5.6.6]
  at org.elasticsearch.action.support.TransportAction.execute(TransportAction.java:84) ~[elasticsearch-5.6.6.jar:5.6.6]
  at org.elasticsearch.client.node.NodeClient.executeLocally(NodeClient.java:83) ~[elasticsearch-5.6.6.jar:5.6.6]
  at org.elasticsearch.client.node.NodeClient.doExecute(NodeClient.java:72) ~[elasticsearch-5.6.6.jar:5.6.6]
  at org.elasticsearch.client.support.AbstractClient.execute(AbstractClient.java:408) ~[elasticsearch-5.6.6.jar:5.6.6]
  at org.elasticsearch.client.support.AbstractClient.index(AbstractClient.java:420) ~[elasticsearch-5.6.6.jar:5.6.6]
  at org.elasticsearch.rest.action.document.RestIndexAction.lambda$prepareRequest$1(RestIndexAction.java:90) ~[elasticsearch-5.6.6.jar:5.6.6]
  at org.elasticsearch.rest.BaseRestHandler.handleRequest(BaseRestHandler.java:80) [elasticsearch-5.6.6.jar:5.6.6]
  at com.floragunn.searchguard.filter.SearchGuardRestFilter$1.handleRequest(SearchGuardRestFilter.java:71) [search-guard-5-5.6.6-18.jar:?]
  at org.elasticsearch.rest.RestController.dispatchRequest(RestController.java:262) [elasticsearch-5.6.6.jar:5.6.6]
  at org.elasticsearch.rest.RestController.dispatchRequest(RestController.java:200) [elasticsearch-5.6.6.jar:5.6.6]
  at com.floragunn.searchguard.ssl.http.netty.ValidatingDispatcher.dispatchRequest(ValidatingDispatcher.java:60) [search-guard-ssl-5.6.6-23.jar:5.6.6-23]
  at org.elasticsearch.http.netty4.Netty4HttpServerTransport.dispatchRequest(Netty4HttpServerTransport.java:505) [transport-netty4-client-5.6.6.jar:5.6.6]
  at org.elasticsearch.http.netty4.Netty4HttpRequestHandler.channelRead0(Netty4HttpRequestHandler.java:80) [transport-netty4-client-5.6.6.jar:5.6.6]
  at io.netty.channel.SimpleChannelInboundHandler.channelRead(SimpleChannelInboundHandler.java:105) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at org.elasticsearch.http.netty4.pipelining.HttpPipeliningHandler.channelRead(HttpPipeliningHandler.java:68) [transport-netty4-client-5.6.6.jar:5.6.6]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102) [netty-codec-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.handler.codec.MessageToMessageCodec.channelRead(MessageToMessageCodec.java:111) [netty-codec-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102) [netty-codec-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:102) [netty-codec-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:310) [netty-codec-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:284) [netty-codec-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:86) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.handler.ssl.SslHandler.unwrap(SslHandler.java:1273) [netty-handler-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.handler.ssl.SslHandler.decode(SslHandler.java:1084) [netty-handler-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.handler.codec.ByteToMessageDecoder.decodeRemovalReentryProtection(ByteToMessageDecoder.java:489) [netty-codec-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.handler.codec.ByteToMessageDecoder.callDecode(ByteToMessageDecoder.java:428) [netty-codec-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:265) [netty-codec-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:340) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1334) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:362) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:348) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:926) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:134) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:644) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.nio.NioEventLoop.processSelectedKeysPlain(NioEventLoop.java:544) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:498) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:458) [netty-transport-4.1.13.Final.jar:4.1.13.Final]
  at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:858) [netty-common-4.1.13.Final.jar:4.1.13.Final]
  at java.lang.Thread.run(Thread.java:748) [?:1.8.0_151]
```

**Comment by jochenkressin** [comment]
Elasticsearch has changed the way index- and delete-operations are handled with 5.3.0:

"Make index and delete operation execute as single bulk item" 
https://github.com/elastic/elasticsearch/pull/22812 

Which means that internally index and delete operations are executed as bulk operations. This means that you need to assign bulk permissions for users as well.

Granting this privilege on cluster level just means the user has the general permission to execute bulk requests. This does not grant anything on index level though. In order to give the user bulk permission for a specific index, you need to add the privilege on index level as well.

The roles shipped with SG already have this permission set, for example the in the INDEX and CLUSTER_COMPOSITE_OPS action group.

The sg_kibana_user has CLUSTER_COMPOSITE_OPS on cluster level, and INDEX permissions for the .kibana index.

**Comment by matejzero** [comment]
Thanks for the information.

Looking at sg_roles from SG project (for 5.6, 6.2 and master), kibana_user has `CLUSTER_COMPOSITE_OPS_RO` permissions.

Anyways, will change to `CLUSTER_COMPOSITE_OPS`, give users permissions and be done with it:)

**Comment by floragunncom** [comment]
Yes, you're totally right, I was looking at the kibana server user role. Thanks for reporting, we will correct this in the next releases! Will reopen until fix is released.

**Comment by jochenkressin** [comment]
Yes, you're totally right, I was looking at the kibana server user role. Thanks for reporting, we will correct this in the next releases! Will reopen until fix is released

**Comment by floragunncom** [comment]
fixed with 6.1.x-22.2 and 6.2.x-22.1

**Comment by VinothKumarKannan** [comment]
Hi All,

Am getting the same issue but in the latest version, anybody can help me

Elastic Version : 6.3.2
SG : 6.x

Error:
**[2018-09-05T04:08:34,703][ERROR][c.f.s.f.SearchGuardFilter] No user found for indices:data/write/bulk from 127.0.0.1:58999 TRANSPORT via netty {_sg_initial_action_class_header=BulkRequest}**

What i wanna do to fix this issue
